Samples/Insights/cpp-win32/InsightsSample.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "InsightsSample.h"
+
+/*
+* Boolean Event
+* 
+* "data": {
+        "value": 1,

```

Comment:
Would be helpful to have comments here saying which part is Part B and which part is Part C. I keep forgetting which one the data value is.